ctapipe/image/toymodel.py

```diff
@@ -40,35 +41,33 @@
 TOYMODEL_RNG = default_rng(0)
 
 
-@u.quantity_input(
-    x=u.m,
-    y=u.m,
-    centroid_x=u.m,
-    centroid_y=u.m,
-    psi=u.deg,
-    time_gradient=u.ns / u.m,
-    time_intercept=u.ns,
-)
+def _obtain_image_no_units(
```

Comment:
Is it really helpful to have this as an extra function? It only ever gets called once

Reply:
This is a leftover from me trying around with `u.quantity_input`